test/integration_test_helper.rb

```diff
@@ -20,18 +20,63 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
-Capybara.register_driver :selenium_chrome do |app|
+
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+
+def get_chrome_options
```

Comment:
Ruby convention is to avoid `get_` in method names
```suggestion
def chrome_options
```

Reply:
Done